courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/2_dataset_api.ipynb

```diff
@@ -151,8 +151,7 @@
     "for i, (x, y) in enumerate(dataset):\n",
```

Comment:
<p>Looking at this again. Do we need EPOCH here? Should it be used as argument in <code>create_dataset</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>